unity-repo-example/.github/workflows/unity-ci.yml

```
@@ -0,0 +1,98 @@
+    - name: Set up .NET
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: '8.0.x'
+        
+    - name: VRChat/Udon compatibility check
+      working-directory: unity-repo-example
+      run: |
+        echo "Checking VRChat/Udon compatibility..."
+        # UdonAnalyzer が実装されている場合はここで実行
+        # 現在はプレースホルダーとして基本的なチェックのみ
+        dotnet build . -c Release -p:DefineConstants="VRCHAT;UDON"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)